src/components/container/SurveyTaskList/SurveyTaskList.stories.tsx

```diff
@@ -10,20 +10,82 @@ export default {
 	}
 };
 
-const render = (args: SurveyTaskListProps) => {
-    return <Layout colorScheme="auto">
-        <SurveyTaskList {...args} />
-    </Layout>;
+// Ease up typing for the 'flattened' sortBehavior props
+const render = (args: any) => {
+	const { 'sortBehavior.type': type, 'sortBehavior.direction': direction, 'sortBehavior.userDefinedOrder': userDefinedOrder, ...rest } = args;
+	const completeArgs: SurveyTaskListProps = {
+		...rest,
+		sortBehavior: {
+			type: type,
+			direction: direction,
+			userDefinedOrder: userDefinedOrder
+		}
+	};
+
+	return (
+		<Layout colorScheme="auto">
+			<SurveyTaskList {...completeArgs} />
+		</Layout>
+	);
 };
 
 export const Incomplete = {
 	args: {
 		limit: 3,
 		status: 'incomplete',
 		title: 'Incomplete Tasks',
-		previewState: "IncompleteTasks",
-		variant: "singleCard",
-		sequential: false
+		previewState: 'IncompleteTasks',
+		variant: 'singleCard',
+		sequential: false,
+		'sortBehavior.type': 'dueDate',
+		'sortBehavior.direction': 'ascending'
+	},
+	argTypes: {
+		sortBehavior: {
+			table: {
+				disable: true
+			}
+		},
+		'sortBehavior.type': {
+			name: 'Sort Type',
+			description: 'The property to sort the incomplete tasks by.',
+			control: {
+				type: 'select'
+			},
+			options: ['alphabetical', 'dueDate', 'insertedDate', 'userDefined', 'shuffle'],
+			table: {
+				category: 'Sort Behavior',
+			}
+		},
+		'sortBehavior.direction': {
+			name: 'Sort Direction',
+			description: 'The direction of the sort.',
+			control: {
+				type: 'radio'
+			},
+			options: ['ascending', 'descending'],
+			if: {
+				arg: 'sortBehavior.type',
+				in: ['alphabetical', 'dueDate', 'insertedDate']
+			},
+			table: {
+				category: 'Sort Behavior',
+			}
+		},
+		'sortBehavior.userDefinedOrder': {
+			name: 'User Defined Order',
+			description: 'An array of survey names in the desired order. Only applies when Sort Type is "userDefined".',
+			control: {
+				type: 'object'
+			},
+			if: {
+				arg: 'sortBehavior.type',
```

Comment:
I'm not sure if it is related to the dot notation or using a string field, but this condition does not appear to be working.  I can't get the user defined order control to show up in the Storybook.